src/pages/index.js

```diff
@@ -1,26 +1,245 @@
+import clientLogo10 from "../files/images/client-logos/client-logo-10.png";
+import clientLogo11 from "../files/images/client-logos/client-logo-11.png";
+import clientLogo12 from "../files/images/client-logos/client-logo-12.png";
+import clientLogo13 from "../files/images/client-logos/client-logo-13.png";
+import clientLogo14 from "../files/images/client-logos/client-logo-14.png";
+import clientLogo15 from "../files/images/client-logos/client-logo-15.png";
+import clientLogo16 from "../files/images/client-logos/client-logo-16.png";
+import clientLogo17 from "../files/images/client-logos/client-logo-17.png";
+import clientLogo18 from "../files/images/client-logos/client-logo-18.png";
+import caseStudyTeaserImg1 from "../files/images/case-study-teasers/case-study-teaser-img-1.png";
+import caseStudyTeaserImg2 from "../files/images/case-study-teasers/case-study-teaser-img-2.png";
+import caseStudyTeaserImg3 from "../files/images/case-study-teasers/case-study-teaser-img-3.png";
+import caseStudyTeaserImg4 from "../files/images/case-study-teasers/case-study-teaser-img-4.png";
+import caseStudyTeaserImg5 from "../files/images/case-study-teasers/case-study-teaser-img-5.png";
+import dataServicesIcon from "../files/icons/data-services-icon.svg";
+import itModernizationIcon from "../files/icons/it-modernization-icon.svg";
+import productDesignIcon from "../files/icons/product-design-icon.svg";
+import securityComplianceIcon from "../files/icons/security-compliance-icon.svg";
+import webCmsIcon from "../files/icons/web-cms-icon.svg";
+import workforceDevelopmentIcon from "../files/icons/workforce-dev-icon.svg";
+import arrowIcon from "../files/icons/arrow-icon.svg";
+import GeneralLayout from "../layouts/general";
 
-// styles
-const pageStyles = {
-  color: "#232129",
-  padding: 96,
-  fontFamily: "-apple-system, Roboto, sans-serif, serif",
-}
-const headingStyles = {
-  marginTop: 0,
-  marginBottom: 64,
-  maxWidth: 320,
-}
-
-const IndexPage = () => {
+const HomePage = ({ data }) => {
   return (
-    <main style={pageStyles}>
-      <title>CivicActions</title>
-      <h1 style={headingStyles}>
-        CivicActions Homesite Redesign
-      </h1>
-    </main>
-  )
-}
+    <GeneralLayout>
+      <section className="home__hero-section">
+        <div className="inner">
+          <img src={homeIntroGraphic} alt="Civic Actions Logo"></img>
+          <div className="info-text">
+            <h2>
+              We help government deliver better public services through modern
+              technology and design.
+            </h2>
+            <h6 className="primary-button">SEE OUR WORK</h6>
+          </div>
+        </div>
+      </section>
+      <section className="home__clients-section">
+        <div className="inner">
+          <div class="body-large">
+            Strategy, design, technology, consulting, and training services for
+            government agencies
+          </div>
+          <div class="grid">
```

Comment:
The requested ticket can be found here https://civicactions-home.atlassian.net/browse/HOMER20-212. It will be worked on this sprint and will be part on a new or just for itself